lib/collection/property-base.js

```diff
@@ -169,6 +169,28 @@ _.assign(PropertyBase.prototype, /** @lends PropertyBase.prototype */ {
         return this && this.__parent && (this.__parent.__parent || this.__parent) || undefined;
     },
 
+    /**
+     * Returns the path of the item
+     *
+     * @returns {Array<string>}
+     */
+    getPath () {
+        const path = [];
+        let item = this;
+
+        while (item) {
+            if (item.name) {
+                path.push(item.name);
+            }
+
+            item = item.__parent;
```

Comment:
Why access private property like this?

Reply:
Because .parent is returning grandparent, and skipping elements.